native/src/main/java/io/xlibb/gateway/exception/ValidationException.java

```diff
@@ -1,29 +1,29 @@
-/*
- * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org). All Rights Reserved.
- *
- * WSO2 LLC. licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-package io.xlibb.gateway.exception;
-
-/**
- * Exception class to represent validation errors.
- */
-public class ValidationException extends Exception {
-
-    public ValidationException(String message) {
-        super(message);
-    }
-}
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org). All Rights Reserved.
```

Comment:
Check other places as well.

```suggestion
 * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org).
```